<!--xml-->
<launch>
<node name="$(env ROS_HOSTNAME)_sample_node" pkg="icarus_rover_v2" type="sample_node"  output="screen" clear_params="true" respawn="true">
    <param name="verbosity_level"       value="NOTICE"/>
    <param name="require_pps_to_start"  value="true"/> 
    <param name="loop1_rate"            value="20.0"/> <!-- Process Update -->  
    <param name="loop2_rate"            value="5.0"/> <!-- Publish -->
</node>
	    <param name="verbosity_level"       value="NOTICE"/> 
</node>
<node name="$(env ROS_HOSTNAME)_snapshot_node" pkg="icarus_rover_v2" type="snapshot_node"  output="screen" clear_params="true" respawn="false"> 
        
    <param name="verbosity_level"       value="NOTICE"/>
        <param name="require_pps_to_start"  value="false"/> 
        <param name="loop1_rate"            value="20.0"/> <!-- Process Update -->  
        <param name="loop2_rate"            value="5.0"/> <!-- Publish -->
        <param name="Mode"                  value="MASTER"/>
</node>
</launch>
